### DISCUSSION TITLE
Does OpenThread support Thread over BLE introduced in Thread 1.3

### DISCUSSION THREAD

**Discussion by FabienFranck** [openthread/openthread#9185] [Q&A] [answered]
Having a look at the code, I see no reference to BLE or ToBLE.
Is the support of ToBLE optional ?
Will OpenThread support ToBLE in the future ?

**Comment by jwhui** [comment]
Thread over BLE is not included in [Thread 1.3 Specification](https://www.threadgroup.org/support#specifications). OpenThread does not currently implement Thread over BLE.

**Reply by FabienFranck** [reply]
Thanks Jonathan.
But in the specification 1.3 there is:
CHAPTER 13 THREAD OVER BLUETOOTH
I checked and Thread over Bluetooth was already in version 1.2.

**Reply by jwhui** [reply, marked as answer]
Thread 1.3.0 published at the link above has "Chapter 13 - Service Registration and Discovery".

The documents you are referring to are internal Thread Group documents.

**Reply by FabienFranck** [reply]
I didn't notice there were a difference between the 'latest/final' internal specification and the 'official' specification.
Thanks for point the official specification.
BTW, is it foreseen to release Thread over Bluetooth in the future ?
I am member of Thread Group, but have not followed latest discussions.
